src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,84 @@
 package core.basesyntax;
 
-public class SalaryInfo {
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class SalaryInfo extends SalaryCalculator {
+    private final SalaryDataParser parser = new SalaryDataParser();
+    private ArrayList<Employee> employees;
+    private LocalDate dayTo;
+    private LocalDate dayFrom;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
```

Comment:
It's good that you try to use oop approach and create different objects. But in this homework it is no needed. Your solution is overcomplicated and difficult to read. Please simplify it. This task can be done in only a single class. Here is the example:
![image](https://user-images.githubusercontent.com/117547096/212723827-7ad34ca4-9e50-4e8a-8825-5611b2550481.png)

Reply:
Please, don't give the code with solution to the students. It looks like cheating :)